### ISSUE TITLE
ERROR: the following arguments are required: --name/--authorization-rule-name

### ISSUE THREAD

**Issue by NathanDotTo** [mspnp/serverless-reference-implementation#126]
When trying:

```
export EVENT_HUB_CONNECTION_STRING=$(az eventhubs eventhub authorization-rule keys list \
     -g $RESOURCEGROUP \
     --eventhub-name $APPNAME-eh  \
     --namespace-name $EH_NAMESPACE \
     -n send \
     --query primaryConnectionString --output tsv)
```
I get:
```
ERROR: the following arguments are required: --name/--authorization-rule-name
```

**Comment by NathanDotTo** [comment]
Should be:

```
export EVENT_HUB_CONNECTION_STRING=$(az eventhubs eventhub authorization-rule keys list \
     -g $RESOURCEGROUP \
     --eventhub-name $APPNAME-eh  \
     --namespace-name $EH_NAMESPACE \
     --name send \
     --query primaryConnectionString --output tsv)
```

It *looks* like `-n` no longer works, so `--name` is required.

**Comment by v-fearam** [comment]
I wanted to inform you that the issue has been addressed in the pull request #125. I will proceed to close the ticket. However, if you encounter any further issues or have additional feedback, please feel free to open a new ticket.
